### DISCUSSION TITLE
Unable to auth against trakt.tv

### DISCUSSION THREAD

**Discussion by zenjabba** [meisnate12/Plex-Meta-Manager#147] [Q&A]
```
| Connecting to Trakt...                                                                             |
| Navigate to: https://trakt.tv/oauth/authorize?client_id=9c8695123e1d2ff7a3dcecf7cf8edd785f77c076858cad59b86a92f7fe06b363&redirect_uri=urn%3Aietf%3Awg%3Aoauth%3A2.0%3Aoob&response_type=code |
| If you get an OAuth error your client_id or client_secret is invalid                               |
| Trakt pin (case insensitive): CFE7D464
| Trakt Error: New Authorization Failed                                                              |
| Trakt Connection Failed
```

I've confirmed my client_id is correct but still getting this error. Tried another users client_id (which is working on there system) still no go

`/usr/bin/docker run --rm -it --name pmm -v /opt/pmm/config:/config:rw -v /mnt:/mnt -v /mnt/unionfs/inbound/librarybase/collections/posters:/config/assets meisnate12/plex-meta-manager:develop --config /config/config.yml --run`

**Comment by meisnate12** [comment]
hey so to start did you have trakt auth working before and now it's failed or this is your first time authenticating?

**Comment by zenjabba** [comment]
First time.

I took a friends auth details (client secret + client id) and it didn’t work.

I took the whole configuration from him in regards to trakt auth and pasted into the config.yml and it works.

**Reply by meisnate12** [reply]
ok, so I reset my auth and was able to reauthenticate just fine. can you have your friend try to reauthenticate with their secret and id and if successful can you have them try and reauthenticate with your secret and id and see if that works. If you're ok enough to send me your id and secret message me privately on [Discord](https://discord.gg/NfH6mGFuAB) and I can test your's myself.

**Comment by zenjabba** [comment]
I have no problems sending you the clientid etc I’ll do it in a few hours to your discord.. Thanks for helping

**Comment by zerodayz1** [comment]
I've got the same issue, using docker image v1.11.3

`[2021-07-29 09:21:11,942] [config.py:241]             [INFO]     | Connecting to Trakt...                                                                             |
[2021-07-29 09:21:11,943] [trakttv.py:46]             [INFO]     | Navigate to: https://trakt.tv/oauth/authorize?client_id=b8d35935debc876b1089b14ee0fd3f5401ffb95d9b7968887525f1e67cce5c25&redirect_uri=urn%3Aietf%3Awg%3Aoauth%3A2.0%3Aoob&response_type=code |
[2021-07-29 09:21:11,943] [trakttv.py:47]             [INFO]     | If you get an OAuth error your client_id or client_secret is invalid                               |
[2021-07-29 09:21:11,943] [config.py:250]             [ERROR]    | Input Failed                                                                                       |
[2021-07-29 09:21:11,943] [config.py:251]             [INFO]     | Trakt Connection Failed `

Have tried existing Trakt apps, and creating new ones, same error on all.

**Reply by meisnate12** [reply]
can you try using the latest develop version

**Reply by zerodayz1** [reply]
Same error, sadly....

`Status: Downloaded newer image for meisnate12/plex-meta-manager:develop
docker.io/meisnate12/plex-meta-manager:develop
|====================================================================================================|
|                                                                                                    |
|        ____  _             __  __      _          __  __                                           |
|       |  _ \| | _____  __ |  \/  | ___| |_ __ _  |  \/  | __ _ _ __   __ _  __ _  ___ _ __         |
|       | |_) | |/ _ \ \/ / | |\/| |/ _ \ __/ _` | | |\/| |/ _` | '_ \ / _` |/ _` |/ _ \ '__|        |
|       |  __/| |  __/>  <  | |  | |  __/ || (_| | | |  | | (_| | | | | (_| | (_| |  __/ |           |
|       |_|   |_|\___/_/\_\ |_|  |_|\___|\__\__,_| |_|  |_|\__,_|_| |_|\__,_|\__, |\___|_|           |
|                                                                            |___/                   |
|           Version: 1.11.3                                                                          |
|====================================================================================================|
|                                            Starting Run                                            |
|====================================================================================================|
| Locating config...                                                                                 |
| Using /config/config.yml as config                                                                 |
|====================================================================================================|
| Using cache database at /config/config.cache                                                       |
|====================================================================================================|
| Connecting to TMDb...                                                                              |
| TMDb Connection Successful                                                                         |
|====================================================================================================|
| Connecting to OMDb...                                                                              |
| OMDb Connection Successful                                                                         |
|====================================================================================================|
| Connecting to Trakt...                                                                             |
| Navigate to: https://trakt.tv/oauth/authorize?client_id=b8d35935debc876b1089b14ee0fd3f5401ffb95d9b7968887525f1e67cce5c25&redirect_uri=urn%3Aietf%3Awg%3Aoauth%3A2.0%3Aoob&response_type=code |
| If you get an OAuth error your client_id or client_secret is invalid                               |
| Trakt pin (case insensitive): | Input Failed                                                                                       |
| Trakt Connection Failed                                                                            |`

**Reply by meisnate12** [reply]
It didn't update the version is Version: 1.11.3 in develop it would be Version: 1.11.3-beta2

**Reply by zerodayz1** [reply]
Good spot...

|           Version: 1.11.3-beta2                                                                    |
|====================================================================================================|
|                                            Starting Run                                            |
|====================================================================================================|
| Locating config...                                                                                 |
| Using /config/config.yml as config                                                                 |
|====================================================================================================|
| Using cache database at /config/config.cache                                                       |
|====================================================================================================|
| Connecting to TMDb...                                                                              |
| TMDb Connection Successful                                                                         |
|====================================================================================================|
| Connecting to OMDb...                                                                              |
| OMDb Connection Successful                                                                         |
|====================================================================================================|
| Connecting to Trakt...                                                                             |
| Navigate to: https://trakt.tv/oauth/authorize?response_type=code&client_id=b8d35935debc876b1089b14ee0fd3f5401ffb95d9b7968887525f1e67cce5c25&redirect_uri=urn%3Aietf%3Awg%3Aoauth%3A2.0%3Aoob |
| If you get an OAuth error your client_id or client_secret is invalid                               |
| Trakt pin (case insensitive): | Input Failed                                                                                       |
| Trakt Connection Failed                                                                            |
|====================================================================================================|
| mal attribute not found                                                                            |
|====================================================================================================|

**Reply by meisnate12** [reply]
So did you go to the links and then get the code and enter it into PMM?

**Reply by zerodayz1** [reply]
Posting for my future self... docker needs to be run in interactive mode. add `-it` to the `docker run` command.
